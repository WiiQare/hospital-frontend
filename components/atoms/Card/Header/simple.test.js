import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import SimpleHeader from "./simple";

describe("SimpleHeader", () => {
  let component;
  beforeEach(() => {
    const res = render(<SimpleHeader />);
    component = res.container;
  });

  it("should render the component", () => {
    expect(component).toMatchSnapshot();
  });
});
