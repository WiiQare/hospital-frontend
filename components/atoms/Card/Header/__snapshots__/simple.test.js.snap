// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleHeader should render the component 1`] = `
<div>
  <div
    class="flex flex-col md:flex-row gap-6 md:gap-0 md:items-center justify-between"
  >
    <div
      class="space-y-2"
    >
      <h2
        class="font-bold text-xl text-gray-700"
      />
      <span
        class="text-xs text-gray-500"
      />
    </div>
  </div>
</div>
`;
